packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ScatterPlot/ScatterPlot.js

```diff
@@ -81,20 +96,35 @@ function ScatterPlot (props) {
       <Background fill={background} />
       <clipPath id={`scatter-plot-${clipPathId}`}>
         <rect
-          height={parentHeight - margin.bottom - margin.top}
-          width={parentWidth - margin.right - margin.left}
+          height={plotHeight}
+          width={plotWidth}
         />
       </clipPath>
       <Group
         clipPath={`url(#scatter-plot-${clipPathId})`}
         left={leftPosition}
         top={topPosition}
       >
+        {tickDirection === 'outer' &&
+          <Background
+            borderColor={colors['dark-5']}
+            fill='#ffffff'
+            height={plotHeight}
+            left={leftPosition}
+            top={topPosition}
+            width={plotWidth}
+          />}
         {dataPoints.map((series, seriesIndex) => {
+          const glyphColor = series.seriesOptions?.color ||
```

Comment:
Is this `series.seriesOptions?.color` value check available in all browsers?

Reply:
No, it's a stage 4 specification, so it's been accepted as a standard, but it's not quite supported by browsers yet. Babel should be handling transpiling this so that it does function.

Reply:
Cool. More info [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). Good to know it'll be functioning with Babel.